contracts/LockDealBundleProvider/LockDealBundleProviderState.sol

```diff
@@ -6,9 +6,5 @@ import "../LockDealNFT/LockDealNFT.sol";
 /// @title LockDealBundleProviderState contract
 /// @notice Contains storage variables
 contract LockDealBundleProviderState {
-    mapping(uint256 => LockDealBundle) public poolIdToLockDealBundle;
-
-    struct LockDealBundle {
-        uint256 firstSubPoolId;
-    }
+    mapping(uint256 => uint256) public poolIdToLockDealBundle;
```

Comment:
I think its better to keep the `first` in the name:
maybe we can try:
`firstPoolIdToBundleId`

Reply:
Fixed with `bundlePoolIdToFirstPoolId`